models/klaviyo__events.sql

```diff
@@ -0,0 +1,73 @@
+with events as (
+
+    {% set exclude_fields = ['campaign_or_flow_partition', 'last_touch_campaign_or_flow_id'] %}
```

Comment:
Just thinking out loud here --- I wonder how the value of excluding these two fields compares to the code readability.  Is it worth it?

Reply:
perhaps fleshing things out more via in line comments would help?